server/server_test.go

```diff
@@ -41,7 +40,6 @@ func (ts *TestSuite) SetupTest() {
 	ts.BaseSetupTest()
 
 	// Load the router & register routes
-	ts.Router = gin.Default()
```

Comment:
Testing result: `server/server_test.go` - these unit tests are failing 

`runtime error: invalid memory address or nil pointer dereference`.

Reply:
Ha, by the way, I found a bug in auth middleware.

Fixed